bench/Chainweb/Pact/Backend/ForkingBench.hs

```diff
@@ -129,24 +129,40 @@ bench rdb = C.bgroup "PactService"
     forkingBench = withResources rdb 10 Quiet
         $ \mainLineBlocks pdb bhdb nonceCounter pactQueue _ ->
             C.bench "forkingBench"  $ C.whnfIO $ do
-              let (T3 _ join1 _) = mainLineBlocks !! 5
+              let (T3 _ join1 pwo1) = mainLineBlocks !! 5
+
+              -- we have to execValidateBlock on `join1` block height to update the parent header
+              void $ validateBlock join1 (payloadWithOutputsToPayloadData pwo1) pactQueue
```

Comment:
You might create a helper function (for the tests only!) like
```haskell
updateParentHeader join pwo queue = void $ validateBlock join (payloadWithOutputsToPayloadData pwo) queue
```
avoiding the comments in the tests and making the tests a bit more expressive.